Nodejs/Product/Nodejs/Repl/InfoReplCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Microsoft.NodejsTools.Repl
+{
+    [Export(typeof(IReplCommand))]
+    internal class InfoReplCommand : IReplCommand
+    {
+        public Task<ExecutionResult> Execute(IReplWindow window, string arguments)
+        {
+            if (window.Evaluator is NodejsReplEvaluator nodeEval)
+            {
+                try
+                {
+                    var nodeExePath = nodeEval.NodeExePath;
+                    if (string.IsNullOrEmpty(nodeExePath))
```

Comment:
IsNullOrWhiteSpace preferred or necessary?

Reply:
no, we probably want to fail in that case, and give the user a clear error message